admin/base/settings/defaults.py

```diff
@@ -25,18 +25,34 @@
     '.osf.io'
 ]
 
+# Email settings. Account created for testing. Password shouldn't be hardcoded
```

Comment:
@cwisecarver I removed the hardcoded password - I figured I should just use a local.py constant